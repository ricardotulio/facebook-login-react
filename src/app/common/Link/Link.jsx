import React from 'react'

const Link = (props) => (
  <a href={props.to}>{props.children}</a>
)

export default Link