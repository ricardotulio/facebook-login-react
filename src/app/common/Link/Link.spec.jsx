import React from 'react'
import renderer from 'react-test-renderer'

import Link from './Link'

test('must render link component', () => {
  const link = renderer.create(
    <Link to="/home">Click Here</Link>
  )
  let tree = link.toJSON()

  expect(tree).toMatchSnapshot()
})