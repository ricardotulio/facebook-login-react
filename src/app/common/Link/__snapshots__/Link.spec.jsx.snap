// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`must render link component 1`] = `
<a
  href="/home"
>
  Click Here
</a>
`;
